.scripts/util.sh

```diff
@@ -38,85 +38,88 @@ copy_modules() {
 	cp "$SRC/sys/xfs/ext2fs/.compile_$TARGET/ext2.xfs" "$MINTDIR"
 	cp "$SRC/sys/xfs/minixfs/.compile_$TARGET/minix.xfs" "$MINTDIR/minix.xfx"
 	cp "$SRC/sys/xfs/nfs/.compile_$TARGET/nfs.xfs" "$MINTDIR"
+	cp "$SRC/sys/xfs/isofs/.compile_$TARGET/isofs.xfs" "$MINTDIR/isofs.xfx"
 }
 
 # modules compatible with all m68k machines (except the FireBee...)
 copy_m68k_modules() {
 	local SYSDIR="$1"
+	local TARGET="$2"
 	mkdir -p "$SYSDIR"
-	cp "$SRC/sys/sockets/xif/asix.xif" "$SYSDIR/asix.xix"
-	cp "$SRC/sys/sockets/xif/picowifi.xif" "$SYSDIR/picowifi.xix"
-	cp "$SRC/sys/sockets/xif/plip.xif" "$SYSDIR"
-	cp "$SRC/sys/xdd/audio/audiodev.xdd" "$SYSDIR"
-	cp "$SRC/sys/xdd/flop-raw/flop_raw.xdd" "$SYSDIR"
+	cp "$SRC/sys/sockets/xif/asix_${TARGET}.xif" "$SYSDIR/asix.xix"
+	cp "$SRC/sys/sockets/xif/picowifi_${TARGET}.xif" "$SYSDIR/picowifi.xix"
+	cp "$SRC/sys/sockets/xif/plip_${TARGET}.xif" "$SYSDIR/plip.xif"
+	cp "$SRC/sys/xdd/audio/.compile_$TARGET/audiodev.xdd" "$SYSDIR"
+	cp "$SRC/sys/xdd/flop-raw/.compile_$TARGET/flop_raw.xdd" "$SYSDIR"
 }
 
 copy_st_modules() {
 	local MCHDIR="$1/st"
 	mkdir -p "$MCHDIR"
 	# TODO: perhaps these four are compatible also with other machines
 	#       but they are awfully old and obsolete so let's keep them here
-	cp "$SRC/sys/sockets/xif/biodma.xif" "$MCHDIR/biodma.xix"
-	cp "$SRC/sys/sockets/xif/de600.xif" "$MCHDIR/de600.xix"
-	cp "$SRC/sys/sockets/xif/dial.xif" "$MCHDIR/dial.xix"
-	cp "$SRC/sys/sockets/xif/pamsdma.xif" "$MCHDIR/pamsdma.xix"
-	cp "$SRC/sys/sockets/xif/rtl8012st.xif" "$MCHDIR/rtl8012st.xix"
-	cp "$SRC/sys/xdd/mfp/mfp.xdd" "$MCHDIR"
+	cp "$SRC/sys/sockets/xif/biodma_000.xif" "$MCHDIR/biodma_000.xix"
```

Comment:
Is there any benefit of keeping the _000 suffix there? (it also breaks 8+3 filename limit)

Reply:
No, just a copy/paste error.  Will fix it.